Ability to create, read, update and delete team members

priyankt68 opened the pull request:
Initial code to
1. Add a new member.
2. Read all the members.
3. Update details of a member.
4. Delete one of more members.

lalitpatel commented:
Hi @priyankt68, we have reviewed your assignment and below are some of the comments from the initial review of the assignment:

- Not all requirements pinned in requirements.txt
- Migration folder is missing
- Deleting a non-existant ID causes a crash
- Used exception classes that haven't been imported
- Wrong content types (it is not JSON) - response to GET request is
    ```python 
    OrderedDict([('userId', 1), ('firstName', 'Bob'), ('lastName', 'Dylan'), ('phone', 783825224), ('role', 'regular')])OrderedDict([('userId', 2), ('firstName', 'Bob'), ('lastName', 'Dylan'), ('phone', 7783825224), ('role', 'regular')])
    ```

priyankt68 commented:
Hi @lalitpatel,

Thanks for reviewing the assignment. I have made changes accordingly and added my comments below.

* Not all requirements pinned in requirements.txt - Fixed. Pinned `djangorestframework==3.11.1` in requirements.txt
* Migration folder is missing - As a practice, I avoid pushing migrations folder. I have added it in .gitignore. In deployments, each developer and  environments(staging or prod) can then have their own sequence of migrations folders. This reduces merge conflicts in my experience.
* Used exception classes that haven't been imported. - Fixed. Imported `Http404` from `django.http`
* Deleting a non-existant ID causes a crash. - Fixed as part of previous bug.
* Wrong content types (it is not JSON) - response to GET request is ... - Fixed. 'jsonised' them using `json.dumps`. (I had realised this before your comment :) )
